### PR TITLE
LPS-119007 Add styles test cases in fi relevant scope

### PR DESCRIPTION
**Pull request by calvinctvo**
### JIRA Ticket:
https://issues.liferay.com/browse/LPS-119007
### Description:
This adds Style Books test cases in frontend-token test.properties
### Test Plan:
1. Add Style Books into front-end token relevant scope
2. Add enable portal.acceptance on smoke tests for StyleBooks.testcase

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/stylebooks/StyleBooks.testcase [hunk 1]
```diff
@@ -124,6 +124,8 @@ definition {
 	@description = "This is for LPS-116078. Can not preview style book when there are no pages in site."
 	@priority = "4"
 	test CannotPreviewStyleBookWhenSiteWithoutPage {
+		property portal.acceptance = "true";
+
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/stylebooks/StyleBooks.testcase:128]: Remove this. Don't need to add more of their tests onto acceptance. We'll let Echo team decide what they want.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/stylebooks/StyleBooks.testcase [hunk 2]
```diff
@@ -241,6 +243,8 @@ definition {
 	@description = "This is for LPS-116106. Page Creator can change style book of content page via Look and Feel settings."
 	@priority = "4"
 	test ChangeStyleBookOfContentPageViaLookAndFeel {
+		property portal.acceptance = "true";
+
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/stylebooks/StyleBooks.testcase:247]: same as above, remove this.

File: modules/apps/frontend-token/test.properties [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+## Test Batch
+##
+
+    #
+    # Relevant
+    #
+
+    test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
+        (portal.acceptance == true) AND \
+        (testray.component.names ~ "Style Books")
```
Comment (john-co) [inline modules/apps/frontend-token/test.properties:18]: use:
testray.main.component.name

Take note of what properties are set in the StyleBooks.testcase.

**Review by john-co — CHANGES_REQUESTED** [review]
Please review comments and resend.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/stylebooks/StyleBooks.testcase (hunk 1 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/stylebooks/StyleBooks.testcase:128]: Add `property testray.component.names = "Frontend Token";` instead and filter this in test.properties

File: modules/apps/frontend-token/test.properties (hunk 3 above)
Comment (john-co) [reply modules/apps/frontend-token/test.properties:18]: use:

(testray.component.name == "Frontend Token") OR \
 (\
(portal.acceptance == true) AND \
(testray.main.component.name ~ "Style Books")
)

**Comment by calvinctvo** [comment]
ci:test:relevant

**Comment by john-co** [comment]
fyi @david-gutierrez-mesa @Tim-Cao , while reviewing this I noticed no test.properties file in `modules/apps/style-book`. Consider adding `ci:test:relevant` coverage for style-book?

**Comment by john-co** [comment]
Just started reviewing :)

File: modules/apps/frontend-token/test.properties [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+##
+## Test Batch
+##
+
+    #
+    # Relevant
+    #
+
+    test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
+        (testray.component.name == "Frontend Token") OR \
```
Comment (john-co) [inline modules/apps/frontend-token/test.properties:17]: testray.component.name**s**

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:stop:relevant

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
ChangeStyleBookOfContentPageViaLookAndFeel and acceptance Style Book tests are included in ci:test:relevant https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1505424086 :heavy_check_mark:

**Comment by Tim-Cao** [comment]
Just start reviewing

**Pull request closed without merging**